File: App/app/src/main/java/com/example/app/DataBase/RegistroDB.java
package com.example.app.DataBase;

import android.provider.BaseColumns;

class RegistroDB
    {
        /**
         * Estructura de la tabla para almacenar toda la información solicitada
         * */
        public static abstract class RegistroEntrada implements BaseColumns{
            //Nombre de la tabla
            public static final String TABLE_NAME = "Registros";

            //Columnas de la tabla

            //Relacionado al usuario
            public static final String ID_USUARIO = "idUsuario";
            public static final String USUARIO = "nombreUsuario";

            //Relacionado al dispositivo
            public static final String DISPOSITIVO = "nombreDispositivo";
            //public static final String TIPO = "tipoDispositivo";
            public static final String MARCA = "marcaDispositivo";
            public static final String SERIE = "serieDispositivo";
            //public static final String DESCRIPCION = "descripcionDispositivo";
        }
    }

File: App/app/src/main/java/com/example/app/DataBase/RegistroDb.kt
package com.example.app.DataBase

import android.provider.BaseColumns

class RegistroDb {
        /**
         * Estructura de la tabla para almacenar toda la información solicitada
         */
        object RegistroEntrada : BaseColumns {
            const val TABLE_NAME = "registro_usuario"
            const val ID_USUARIO = "idUsuario"
            const val USUARIO = "nombreUsuario"
            const val DISPOSITIVO = "nombreDispositivo"
            const val MARCA = "marcaDispositivo"
            const val SERIE = "serieDispositivo"
            const val DESCRIPCION = "descripcionDispositivo"
        }
}


